kineverse/OperationCall[] operations
---
bool   success
string error_msg